forge-game/src/main/java/forge/game/spellability/StackItemView.java

```diff
@@ -66,6 +67,8 @@ void updateOptionalCost(SpellAbilityStackInstance si) {
             if (cost == OptionalCost.Kicker1 || cost == OptionalCost.Kicker2)
                 kicked = true;
             if (cost == OptionalCost.Entwine)
+                offspring = true;
+            if (cost == OptionalCost.Offspring)
```

Comment:
switch booleans

Reply:
imo we should look to rewrite updateOptionalCost in a later day